drivers/filesystems/udfs/create.cpp

```diff
@@ -838,6 +849,7 @@ UDFCommonCreate(
             //  the I/O Manager will perform checks on the FSD's behalf)
             if (!(RelatedObjectName.Length) || (RelatedObjectName.Buffer[0] != L'\\')) {
                 AdPrint(("    Wrong pathname (1)\n"));
+                ReturnedInformation = FILE_DOES_NOT_EXIST;
```

Comment:
Not sure.

Reply:
I wasn't sure about using ``FILE_DOES_NOT_EXIST``. I posted a comment about the same in the development channel. Will make this and the rest return 0.